integration_tests/integration/referral.spec.js

```diff
@@ -441,23 +441,23 @@ describe('Referral form', () => {
           `/referrals/${draftReferral.id}/service-category/428ee70f-3001-4399-95a6-ad25eaaede16/desired-outcomes`
         )
 
-      cy.contains('Accommodation completion date')
+      cy.contains('Enforceable days')
```

Comment:
Out of curiosity, does swapping this have an effect? I don't know the matcher syntax here but it looks like it doesn't actually make any sequence assertions on top level

Reply:
No, it does not have an effect; however I thought it best to change the order as it's more clear to the reader/dev the order in how it should appear on the page.

Reply:
Thanks!